manifests/r/RCharan/wiki/1.0/RCharan.wiki.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: RCharan.wiki
+PackageVersion: "1.0"
+PackageLocale: en-US
+Publisher: R Charan
+PackageName: wiki
+License: MIT License

```

```suggestion
License: Apache-2.0
```
I'm not a lawyer, just copying from the project page. If this shouldn't change, then please feel free to ignore the suggestion.